MedicalApp.BuisnessServices/Mappers/Implementation/PatientMapper.cs

```diff
@@ -30,7 +41,15 @@ public void MapFromEntity(PatientEntity item, Patient targetItem)
                 targetItem.FirstName = item.FirstName;
                 targetItem.LastName = item.LastName;
                 targetItem.Age = item.Age;
-                targetItem.RoomId = item.RoomId;
+                //targetItem.RoomId = item.RoomId;
+                if (item.Room != null)
+                {
+                    if(targetItem.Room == null)
+                    {
+                        targetItem.Room = new Room();
+                    }
+                    roomMapper.MapFromEntity(item.Room, targetItem.Room);
```

Comment:
It will be better if you will write something like this:
`targetItem.Room ?? new Room()`

Reply:
The expression was changed.